integration_tests/commands/http/command_getkeys_test.go

```diff
@@ -0,0 +1,30 @@
+package http
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetKeys(t *testing.T) {
```

Comment:
Please use the same set of tests that exists for async

Reply:
Yes, I have added  the same set of tests that exists for async